client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -193,14 +195,28 @@ class EmailProvidersComparison extends React.Component {
 					},
 					comment: '{{price/}} is the formatted price, e.g. $20',
 			  } );
+		const providerTitle = config.isEnabled( 'titan/phase-2' )
+			? translate( 'Email' )
+			: translate( 'Titan Mail' );
+		const providerCtaText = config.isEnabled( 'titan/phase-2' )
+			? translate( 'Add Email' )
+			: translate( 'Add Titan Mail' );
```

Comment:
I believe we could simplify this code by taking advantage of `titanMailProduct` which should have the right product name.

Reply:
I have simplified the code, but the product name may not match the name we're using for non-integrated sales, so I am using the new helper you suggested I use!